foreman/data_refinery_foreman/foreman/main.py

```diff
@@ -1321,6 +1318,7 @@ def cleanup_the_queue():
         jobs = nomad_client.jobs.get_jobs()
     except:
         # If we cannot reach Nomad now then we can wait until a later loop.
+        logger.exception("Couldn't query Nomad about current jobs.")
```

Comment:
Can you make this `warn` with `exc_info=1` so that we log the exception but it doesn't go to Sentry?

Reply:
sure!

```suggestion
        logger.exception("Couldn't query Nomad about current jobs.", exc_info=1)
```